Jenkinsfile_nightly

```diff
@@ -1,16 +1,16 @@
 #!groovy
 
-properties(
+properties([
     // H allow predefined but random minute see https://en.wikipedia.org/wiki/Cron#Non-standard_characters
     pipelineTriggers([cron('H 05 * * *')])
-)
+])
 
 @Library("Infrastructure")
 
 def type = "angular"
 def product = "ccd"
 def app = "case-management-web"
 
-withPipeline(type, product, app) {
+withNightlyPipeline(type, product, component) {
```

Comment:
`app` was correct, as defined above no? `component` would be undefined

Reply:
Not having a great day with that one!